### PR TITLE
Add plugin-check page

### PR DESCRIPTION
**Pull request by afstarosta**

### CODE REVIEW

**Comment by afstarosta** [comment]
`index.php?route=payment/ebanx_plugincheck`

File: catalog/controller/payment/ebanx_plugincheck.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?php
+
+class ControllerPaymentEbanxPlugincheck extends Controller {
+	const EBANX_VERSION = '2.4.0';
```
Comment (Ruanito) [inline catalog/controller/payment/ebanx_plugincheck.php]: It's better call `PLUGIN_VERSION` and set as private

File: catalog/controller/payment/ebanx_plugincheck.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+	public function index() {
+		$info_list = array(
+			'mysql'          => $this->getSQLVersion(),
```
Comment (Ruanito) [inline catalog/controller/payment/ebanx_plugincheck.php]: Open cart only use mysql?

File: catalog/controller/payment/ebanx_plugincheck.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+	private function getSQLVersion()	{
+		$query = $this->db->query('SELECT version() AS version');
+		return $query->row['version'];
+	}
+
+	private function getConfigs()	{
+		$configs = array();
+
+		if(!is_null($this->config->get('ebanx_express_mode'))){
```
Comment (Ruanito) [inline catalog/controller/payment/ebanx_plugincheck.php]: You should create a function to validated if config is null
```suggestion
                if (self::isEbanxExpressMode()) 
```
```
private static function isEbanxExpressMode() {
		    return !is_null($this->config->get('ebanx_express_mode'));
}
```

File: catalog/controller/payment/ebanx_plugincheck.php [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+			$configs['ebanx_express'] = array(
+				'ebanx_express_status' => $this->config->get('ebanx_express_status') === 1,
+				'ebanx_express_mode' => $this->config->get('ebanx_express_mode'),
+				'ebanx_express_enable_installments' => $this->config->get('ebanx_express_enable_installments') === 1,
+				'ebanx_express_max_installments' => $this->config->get('ebanx_express_max_installments'),
+				'entry_installments_interest' => $this->config->get('ebanx_express_installments_interest'),
+			);
+		}
+
+		if(!is_null($this->config->get('ebanx_mode'))){
```
Comment (Ruanito) [inline catalog/controller/payment/ebanx_plugincheck.php]: You should use  a function here too

File: catalog/controller/payment/ebanx_plugincheck.php [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+	private function getSQLVersion()	{
+		$query = $this->db->query('SELECT version() AS version');
+		return $query->row['version'];
+	}
+
+	private function getConfigs()	{
+		$configs = array();
+
+		if(!is_null($this->config->get('ebanx_express_mode'))){
+			$configs['ebanx_express'] = array(
```
Comment (Ruanito) [inline catalog/controller/payment/ebanx_plugincheck.php]: You should use a function just to return the array with ebanx_express array

File: catalog/controller/payment/ebanx_plugincheck.php [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+				'ebanx_express_status' => $this->config->get('ebanx_express_status') === 1,
+				'ebanx_express_mode' => $this->config->get('ebanx_express_mode'),
+				'ebanx_express_enable_installments' => $this->config->get('ebanx_express_enable_installments') === 1,
+				'ebanx_express_max_installments' => $this->config->get('ebanx_express_max_installments'),
+				'entry_installments_interest' => $this->config->get('ebanx_express_installments_interest'),
+			);
+		}
+
+		if(!is_null($this->config->get('ebanx_mode'))){
+			$configs['ebanx'] = array(
```
Comment (Ruanito) [inline catalog/controller/payment/ebanx_plugincheck.php]: You should create a function to return a ebanx config

**Review by Ruanito — CHANGES_REQUESTED** [review]

**Review by Ruanito — APPROVED** [review]

**Pull request merged**